packages/swarm-components/src/Button.test.jsx

```diff
@@ -45,5 +45,48 @@ describe('👀 components are visually the same', function() {
 				await renderer.screenshot(<Button primary>Must press me</Button>)
 			).toMatchImageSnapshot();
 		});
+
+		it('Bordered', async () => {
+			expect(
+				await renderer.screenshot(<Button bordered>Press me</Button>)
```

Comment:
Not sure if it's been talked about much, but for performance and boilerplate reasons it would be pretty nice to see if we could compose one-screenshot-per-test-suite rather than one screenshot per prop.

Short of that, I'd strongly recommend using [`test.each`](https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout) from Jest to write these tests in a way that is _much_ easier to read and maintain. This is just freehand code, but it should look something like

```jsx
const testCases = [
  ['Bordered', <Button bordered>Bordered</Button>],
  ['Icon with text', <Button icon="bolt">Icon with text</Button>],
  ...
  ['Small', <Button small>Small</Button>],
];

test.each(testCases('%s', async (descr, element) => {
  expect(await renderer.screenshot(element)).toMatchImageSnapshot()
});
```

Jest will take care of providing fine-grained test case error messages.